content/Documentation Guidelines - Sebastián Caraballo.md

```diff
@@ -0,0 +1,112 @@
+```
+
+---
+
+#### **Postman**
+
+![image](images/postman.jpg)
+
+[Postman](https://www.postman.com/) is an amazing API client we use locally or with our testing environments so everyone on the team is able to inspect endpoint responses and debug. This is especially useful for frontend developers since it enables them to quickly get responses based on the request attributes. Furthermore, the environment can be easily switched in order to make a request (e.g. `staging`, `production`, and so on). Postman helps the team to better understand how the API works.
+
+---
+
+#### **Repository READMEs**
+[READMEs](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes) is a powerful yet simple way to communicate to your team developers about important project settings. Here is an [example](https://github.com/rootstrap/rails_api_base) from our Rails API Template. There are plenty of aspects that may be included, but it's important that developers take ownership of this initiative and update this file whenever is needed. Besides keeping the README file updated on setting up the project for development and listing used Ruby gems, we also started listing all the environment variables names needed for the different environments to work correctly.
+
+---
+
+#### **ActiveAdmin Dashboard**
+[ActiveAdmin](https://activeadmin.info/) is a popular administration framework for Ruby on Rails applications. This plugin makes implementing interfaces for managing content simple. Not only does it offer an extensible way to customize all different sections, but it also provides the user with a dashboard. So, as mentioned before about documenting different aspects of the administration site, we added these links on a new *Documentation panel*.
+
+---
+
+#### **Notion**
+
+![image](images/notion-template-05.png)
+
+[Notion](https://www.notion.so/product) is an all-in-one platform that connects all the company areas: from product & design, to engineering. It allows us to share knowledge about projects, internal initiatives and so much more. In this specific project, designers portrayed content guidelines so that both web & mobile versions keep visual consistency. As an example, designers detailed some image upload specifications. By doing this, the content manager is able to adapt content so that it fits perfectly on the site. This is just one example of the different capabilities this tool offers.
+
+---
+
+### **Conclusions**
+
+I strongly believe good coding practices should be accompanied by high-quality documentation all along the development cycle. It helps new team members onboard faster and understand the project right away. It helps designers to understand content architecture so they can reason on it. It helps clients to communicate stakeholders about product capabilities. It helps developers to be aware of the whole system. In conclusion, **documenting increases product quality and reduces development time in the long run.**
```

Comment:
```suggestion
I strongly believe that good coding practices should be accompanied by high-quality documentation throughout the development cycle. It helps new team members get onboarded faster and understand the project from the get-go. It also helps designers to understand content architecture so they can reason on it. Additionally, it helps clients to communicate with stakeholders about product capabilities. It can also help developers to be more aware of the whole system. In conclusion, **documenting increases product quality and reduces development time in the long run.**
```